openslides/core/static/js/core/core.js

```diff
@@ -606,6 +633,55 @@ angular.module('OpenSlidesApp.core.site', [
             }
         );
     };
+
+    // *** countdown ***
+    $scope.addCountdown = function () {
+        var defaultvalue = parseInt(Config.get('projector_default_countdown').value);
+        $http.post('/rest/core/projector/1/activate_elements/', [{
+                name: 'core/countdown',
+                status: 'stop',
+                visible: true,
+                countdown_time: defaultvalue,
+                default: defaultvalue,
+                stable: false
+        }]);
+    };
+    $scope.removeCountdown = function (countdown) {
+        $http.post('/rest/core/projector/1/deactivate_elements/', [countdown.uuid]);
+    };
+    $scope.showCountdown = function (visible, uuid) {
+        var data = {};
+        data[uuid] = {
+            "visible": !visible
+        };
+        $http.post('/rest/core/projector/1/update_elements/', data);
+    };
+    $scope.startCountdown = function (countdown) {
+        var data = {};
+        var serverOffset = Date.parse(new Date().toUTCString()) - countdown.context.server_time;
+        var time = Date.now() - serverOffset + countdown.countdown_time;
```

Comment:
You must not calculate the server offset if someone clicks start but immendiatelly after a new projector element comes in via autoupdate or after initial loading.